test/specs/sorting.js

```diff
@@ -0,0 +1,42 @@
+const { pages } = require('../pages/Pages');
+const chai_expect = require('chai').expect;
+var _ = require('lodash');
+
+describe('Sorting on Inventory Page', () => {
+    beforeEach( async () => {
+        await pages.loginPage.navigate();
+        await pages.loginPage.performLogin('standard_user', 'secret_sauce');
+    });
+
+    it ("Verify sorting by name from Z to A", async () => {
+        let sortingZA = _.reverse(await pages.inventoryPage.getAllNames());
```

Comment:
What if you will have an array of `tests`. Where each test is an object and each object has methods: `performSorting`, and `verifySortedData` and a property `sortingOrder` (e.g. "price ascending", "name descending", etc). Every method will use pages.inventoryPage methods to perform all the necessary actions

If you will have e such an array than you will be able to write smth like:
`tests.forEach(test => it ("Verify sorting by " + test.sortingOrder, async () => {` and use test.verifySortedData and test.performSorting methods inside of the it function